breakdown.go

```diff
@@ -156,27 +153,16 @@ func (k breakdownMetricsKey) hash() uint64 {
 
 // breakdownTiming holds breakdown metrics.
 type breakdownTiming struct {
-	// transaction holds the "transaction.duration" metric values.
-	transaction spanTiming
-
-	// Padding to ensure the span field below is 64-bit aligned.
-	_ pad32
-
 	// span holds the "span.self_time" metric values.
 	span spanTiming
 
-	// breakdownCount records the number of transactions for which we
-	// have calculated breakdown metrics. If breakdown metrics are
-	// enabled, this will be equal transaction.count.
-	breakdownCount uintptr
+	// Padding to ensure the span field below is 64-bit aligned.
```

Comment:
I tried removing the padding, but on 32-bit architecture the `uintptr` is 4 bytes rather than 8 (64-bit), making the total struct size 12 bytes in 32-bit architectures instead of the 16 that we get by default on 32-bit architectures.

We could arguably change the implementation to always use an `uint64`, which gives us a maximum value of `18,446,744,073,709,551,615` which is quite large already, not sure what benefit `uintptr` would give us since we're not using it for pointer references anyway.

I think switching to uint64 makes sense since instead of allocating padding bytes in memory, we'd use them for the actual count rather than having padding on 32-bit architectures, I'll make the change, but re-request a review from you to make sure we're ok with `uint64` vs `uintptr`.

Reply:
Looks sensible, and simpler - thanks.